package.json

```diff
@@ -14,6 +14,8 @@
     "lint-staged": "^10.2.6",
     "markdown-it-deflist": "^2.0.3",
     "markdown-it-footnote": "^3.0.2",
+    "markdown-it-image-lazy-loading": "^1.0.2",
+    "markdown-it-imsize": "^2.0.1",
```

Comment:
Does `markdown-it-imsize` also _optimize_ images?

Reply:
no, just allows for width and height props

Reply:
Nope, it just allows you to set default display sizes. Which helps page load time and helps lower [CLS](https://web.dev/cls/).